src/components/Molecules/navigation/NavBar.css

```diff
@@ -0,0 +1,22 @@
+.nav-bar {
+    font-family: Helvetica, Garamond, Arial;
+    background-color: hsl(36, 100%, 90%);
+    background-size: cover;
+    background-attachment: fixed;
+    position: relative;
```

Comment:
Why? Is er een ander element dat absoluut gepositioneerd is ten opzichte van de nav-bar?